src/anemoi/training/train/train.py

```diff
@@ -289,11 +314,12 @@ def _log_information(self) -> None:
         LOGGER.debug("Total number of auxiliary variables: %d", len(self.config.data.forcing))
 
         # Log learning rate multiplier when running single-node, multi-GPU and/or multi-node
-        total_number_of_model_instances = (
+        total_number_of_model_instances = int(
```

Comment:
This is separate from transfer learning, no?

Reply:
Removing this!